src/main/java/htsjdk/variant/vcf/VCFHeaderLineTranslator.java

```diff
@@ -48,13 +50,19 @@
     }
 
     public static Map<String,String> parseLine(VCFHeaderVersion version, String valueLine, List<String> expectedTagOrder) {
-        return mapping.get(version).parseLine(valueLine,expectedTagOrder);
+        return mapping.get(version).parseLine(valueLine,expectedTagOrder,Collections.emptySet());
+    }
+    
+    public static Map<String,String> parseLine(VCFHeaderVersion version, String valueLine, List<String> expectedTagOrder, Set<String> optionalTags) {
+        return mapping.get(version).parseLine(valueLine,expectedTagOrder,optionalTags);
     }
 }
 
 
 interface VCFLineParser {
     public Map<String,String> parseLine(String valueLine, List<String> expectedTagOrder);
+
+    public Map<String,String> parseLine(String valueLine, List<String> expectedTagOrder, Set<String> optionalTags);
```

Comment:
Note that this will break any code that currently subclasses `VCFLineParser` so it is a breaking API change

Reply:
How can I possibly solve this?